.github/workflows/ci.yml

```diff
@@ -23,7 +23,7 @@ on:  # yamllint disable-line rule:truthy
   push:
     branches: ['v[0-9]+-[0-9]+-test']
   pull_request:
-    branches: ['main', 'v[0-9]+-[0-9]+-test']
+    branches: ['main', 'v[0-9]+-[0-9]+-test', 'v[0-9]+-[0-9]+-stable']
```

Comment:
All stable tags or just 2.10?

Reply:
All. We don't want to make special exceptions.